test/flanders/markdown_test.clj

```diff
@@ -28,11 +28,11 @@ description text
 * This entry is required
 
 * Keyword Key
-  * Plumatic Schema: `\"type\"`
+  * Plumatic Schema: `type`
```

Comment:
In the markdown, I imagined that we'd show both the valid plumatic schema and a valid JSON schema.